### ISSUE TITLE
Fix usb name 

### ISSUE THREAD

**Issue by rupraph** [rupraph/pipo-modules-FW#459]
On midi usb, shows as espressif

**Comment by rupraph** [comment]
PR #471 fix was not enough and more testing showed name was only a smoking gun. Issue is Midi enumeration.

Midi enumeration does not succeed systematically when plugged in (both Mac and Windows. works almost always on board reset). 
Might be a USB init timing issue.
(tried on range to remove CDC at boot, board does not boot...)

+motion not detected after switch to Midi mode ? (but was in OSC ??)

**Comment by rupraph** [comment]
-> solved by adding TinyUSBDevice.detach() before MidiUsb.begin() then attach()
